### ISSUE TITLE
Add aria-label="clear selection" to p-dropdown's clear icon

### ISSUE THREAD

**Issue by ajsaraujo** [primefaces/primeng#11378]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => Search github for a similar issue or PR before submitting
[X] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Current behavior**
p-dropdown's clear button does not have an aria-label describing it, it would be nice having one for accessibility reasons. What would be even better would be turning it into a button element, achieving better semantics - currently, it is an icon.

* **Angular version:** 13
* **PrimeNG version:** 13
* **Browser:** Chrome

**Comment by cetincakiroglu** [comment]
Hi,

We're fully focused on Accessibility. Thanks so much for sharing these with us.
Our main goal is to consider all the components one by one and document the individual accessibility features. In doing so, it is to correct the deficiencies or faulty implementations of that component. For this reason, our team carries out a comprehensive and meticulous study. Of course, it will take some time to complete this process.
Currently, working on the components specified in your attached documentation will indeed be a temporary solution for us and we cannot guarantee that these solutions will not change. Because we examine a lot of accessibility documents and try to make the best decisions for users. At this stage, the best decision that can be made about such issues is to stop the actions to be taken on them for the time being. We will reactivate these issues after we complete our work on accessibility in Prime* component libraries. In this way, we will be able to make better decisions about issues.

Details; https://www.primefaces.org/primeng/accessibility

Thanks a lot for your understanding! 

Best Regards
